packages/calling/src/CallingClient/line/index.ts

```diff
@@ -225,7 +230,7 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
    * Initiates a call to the specified destination.
    * @param dest - The call details including destination information.
    */
-  public makeCall = (dest: CallDetails): ICall | undefined => {
+  public makeCall = (dest?: CallDetails): ICall | undefined => {
```

Comment:
Nit: Not sure if we need to make this an optional. We can try and check for the empty object itself. Will check and make required changes based on the manual testing.